### PR TITLE
q 2.0.19

### PR DESCRIPTION
**Pull request by harelba**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/q.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Q < Formula
 
   desc "Run SQL directly on CSV or TSV files"
   homepage "https://harelba.github.io/q/"
-  url "https://github.com/harelba/q/archive/2.0.18.tar.gz"
-  sha256 "db4f90602d3c45395675fa09b94293078f0ff693a94ec5fb23ae7041492fcebd"
+  url "https://github.com/harelba/q/archive/2.0.19.tar.gz"
+  sha256 "cd4c60923bc40f53d974b54849f76096bf9901407c618cd0a3ccbc322aacc97d"
   license "GPL-3.0-only"
```
Comment (chenrui333) [inline Formula/q.rb:8]: @harelba this should be `GPL-3.0-or-later`, right? (as I read thru the file header)

**Comment by chenrui333** [comment]
@harelba thanks for the PR, great to have you confirm some license information. :)

File: Formula/q.rb (hunk 1 above)
Comment (harelba) [reply Formula/q.rb:8]: true. Fixing. thanks.

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @harelba  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by harelba** [comment]
The test failures seem like some global issue, not something directly related to the PR itself, right?

**Comment by SMillerDev** [comment]
Yeah, restarted them

**Comment by harelba** [comment]
would you mind restarting again? one of the tests seems to fail on the same issue

**Comment by harelba** [comment]
sorry for bothering with this, but is possible to restart the jobs again? it seems there was some git issue there

**Comment by chenrui333** [comment]
Thanks @harelba!

**Pull request closed without merging**